### DISCUSSION TITLE
What's the point of downloading Quarto if RStudio is installed?

### DISCUSSION THREAD

**Discussion by urx449** [quarto-dev/quarto-cli#3852] [Q&A] [answered]
According to the Quarto website, the first step is to download and install Quarto. If I have RStudio, is this necessary? Quarto in RStudio seems to work without installing anything extra.

**Comment by harrelfe** [comment, marked as answer]
I've been wondering the same thing.  The system makes this opaque.  But the main reason to install Quarto yourself is that Quarto is still quite fluid, with a number of bugs that have been corrected in the development version but not in the production version used by RStudio.

**Reply by dragonstyle** [reply]
This is just the reason we recommend it - we do our best to align Quarto releases with RStudio releases so RStudio contains the newest stable version of Quarto, but we can't always align those, so there may be a newer stable Quarto release with fixes and new features. Since we tend to coordinate our documentation to the the latest stable release, we always recommend that you start by installing that latest stable version.
